tests/trainer/logging/test_logger_connector.py

```diff
@@ -330,7 +332,7 @@ def test_call_back_validator(tmpdir):
     ]
 
     assert (
-        funcs_name == callbacks_func
+        sorted(funcs_name) == sorted(callbacks_func)
```

Comment:
I believe funcs_name is already sorted

```suggestion
        funcs_name == sorted(callbacks_func)
```